src/app/test/router.ts

```diff
@@ -12,11 +12,15 @@ export function testRouter() {
   const router = express.Router()
 
   router.post('/run-tests', RunTestsSchema, async (req, res) => {
-    const testSession = createTestSession(req.body)
+    try {
+      const testSession = createTestSession(req.body)
 
-    const result = await runTests(testSession)
+      const result = await runTests(testSession)
 
-    return res.json(result.toTestResponse())
+      return res.json(result.toTestResponse())
+    } catch (e) {
+      res.status(500).send({ reason: e.message })
```

Comment:
1. Shouldn't it call `next(e)` so that the error is passed to error handler?
2. We need to cleanup test sessions just in case, otherwise, if the error was thrown while tests were running, the tested hostname will become soft-locked for future tests until server is restarted - take a look at: https://github.com/fingerprintjs/dx-team-mock-for-proxy-integrations-e2e-tests/blob/88bbdbde69b615ff6f891d361447ad670c674f68/src/app/test/service/session.ts#L50-L67

Reply:
1- I wanted format the response of error, so I can handle it on frontend.
2- How can I cleanup a test session that throws an error?

Reply:
1: But right the `catch {}` block does the same thing as the generic error handler: 
```ts
function errorHandler(err: Error, req: Request, res: Response, _: NextFunction) {
  console.error(err)
  res.status(500).send({ reason: err.message })
}
```

2. You need to remove the created test session from `testSessions` map.

Reply:
2- Can you check again, is this what you expect?